app/footer.tsx

```diff
@@ -1,3 +1,176 @@
+import Image from 'next/image';
+import { CiGlobe } from 'react-icons/ci';
+import { FaFacebookSquare, FaLinkedinIn, FaTwitter } from 'react-icons/fa';
+import { MdMailOutline, MdOutlineCall } from 'react-icons/md';
+import { SlLocationPin } from 'react-icons/sl';
+
 export default function Footer() {
-  return <></>;
+  const items = {
+    quickLinks: [
+      { text: 'Telephone Directory', link: '' },
+      { text: 'Books and e-library', link: '' },
+      { text: 'Important Links', link: '' },
+      { text: 'Downloads', link: '' },
+      { text: 'RTI', link: '' },
+      { text: 'NIT KKR IT Infrastructure Usage Policy', link: '' },
+      { text: 'NIT KKR @NDL', link: '' },
+      { text: 'Telephone Directory', link: '' },
+    ],
+    about: [
+      { text: 'Council of NITs', link: '' },
+      { text: 'NAD Digilocker', link: '' },
+      { text: 'NIRF (Data & Certificate)', link: '' },
+      { text: 'NBA Accreditation Status', link: '' },
+      { text: 'ARIIA', link: '' },
+      { text: 'Skill Hub (PMKVY 4.0)', link: '' },
+      { text: 'Jobs @ NIT KKR', link: '' },
+    ],
+    departments: [
+      { text: 'Vigilance Corner', link: '' },
+      { text: 'PRO', link: '' },
+      { text: 'OBC & PWD Cell', link: '' },
+      { text: 'SC/ST Cell', link: '' },
+      { text: 'Join Alumni Association', link: '' },
+      { text: 'User Login', link: '' },
+      { text: 'Contact Us', link: '' },
+    ],
+  };
+
+  return (
+    <>
+      <section className="flex flex-col justify-center bg-[#000] md:flex-row lg:items-center">
+        <blockquote className="items flex flex-col items-center justify-center px-10">
+          <Image
+            src="https://s3-alpha-sig.figma.com/img/18b7/a13d/8bbb852e070e69b3de2a5ac59d20f501?Expires=1707696000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=gtaYqKeU8izdKAOskR5FvCKz15ZM9I~Fai49G9uVnIM1GLBs1e0sYvodH7rpjGkPIJ2yOczgOM6i~jECdaO3ufZHCUY~~80goh86AFkw~6vyJmKTmFVNUlz5kHfqMFmuaPqjTuoJY7XeFeDdYHTjrvGGNe6ATas9-IYPC2gSKavFZ8L5~tNY1vn-~IDnjdeemmKPOsyWbKDmUJWTfI~s~wsWLnKiwVFYpPjYZtwj-u1-~Dy2SuSFOKALF-AnZk1oSBIJrwnSS~Ox6aAP64YsclxQDO6NyJGWBbZP3s5jgVh6tHrHSzR9irkt9nyZj0ndSptaXYTRx4QFx8MYRfeDdw__"
+            alt="NIT KURUKSHETRA LOGO"
+            className="pt-5"
+            width={66}
+            height={66}
+          />
+          <header className="font-display max-w-[250px] pt-3 text-center font-serif text-[22px] text-[#fff]">
+            National Institute of Technology, Kurukshetra
+          </header>
+          <aside className="max-w-[250px] pt-3 text-center font-sans text-[16px] text-[#8B949E]">
+            WRXF+23G, NIT, Thanesar, Haryana 136119
+          </aside>
+
+          <object>
```

Comment:
Okay just what. Did you even read the docs. Looks like you've just asked ChatGPT to translate this.

> The <object> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents an external resource

"external resource"